okio/src/main/java/okio/BufferedSource.java

```diff
@@ -499,4 +500,10 @@ boolean rangeEquals(long offset, ByteString bytes, int bytesOffset, int byteCoun
 
   /** Returns an input stream that reads from this source. */
   InputStream inputStream();
+
+  /** Returns a Reader that reads from this source. */
```

Comment:
I don’t think I like this overload. When would a user not have a charset? As far as I’m concerned not having a charset is a big bug in how you think about your data.